src/librustdoc/formats/renderer.rs

```diff
@@ -9,7 +9,7 @@ use crate::formats::cache::Cache;
 /// Allows for different backends to rustdoc to be used with the `run_format()` function. Each
 /// backend renderer has hooks for initialization, documenting an item, entering and exiting a
 /// module, and cleanup/finalizing output.
-crate trait FormatRenderer<'tcx>: Clone {
+crate trait FormatRenderer<'tcx>: Sized {
```

Comment:
Why is this bound necessary?

Reply:
This is still waiting on an answer.

Reply:
This change was removed because I wasn't able to get rid of the cloning here. (I'm going to attempt that locally and see if I can get the cloning working in a new PR.)

But the reason I made the change before was because one of the methods returns `Self` without indirection.